pandas/tests/io/json/test_pandas.py

```diff
@@ -651,11 +649,9 @@ def test_series_roundtrip_empty(self, orient):
         data = empty_series.to_json(orient=orient)
         result = read_json(data, typ="series", orient=orient)
 
-        expected = empty_series
-        if orient in ("values", "records"):
-            expected = expected.reset_index(drop=True)
-        else:
-            expected.index = expected.index.astype(float)
+        expected = empty_series.reset_index(drop=True)
+        if orient in ("split"):
+            expected.index = expected.index.astype(np.float64)
```

Comment:
any reason to change `float` to `np.float64`? (not an issue, just trying to understand)

Reply:
No reason, my fingers must have liked np.float64 better than float in this case :-)